sdk/eventhub/event-hubs/src/util/parseEndpoint.ts

```diff
@@ -8,7 +8,7 @@
  * @internal
  */
 export function parseEndpoint(endpoint: string): { host: string; hostname: string; port?: string } {
-  const hostMatch = endpoint.match(/.*:\/\/([^\/]*)/i);
+  const hostMatch = endpoint.match(/.*:\/\/([^/]*)/i);
```

Comment:
Hmm.. doesn't make a difference, right?

Reply:
Right. ESLint was complaining because that backslash was unnecessary.